src/main/java/swbot/Task.java

```diff
@@ -0,0 +1,51 @@
+package swbot;
```

Comment:
Perhaps refactor `Task` and other related classes like `Deadline`, `Event` and `ToDo` into a `swbot.task` package?